src/HDFFile.cpp

```diff
@@ -242,7 +242,8 @@ void writeAttributes(hdf5::node::Node const &Node, nlohmann::json const *Value,
 /// \param JsonValue    json value array of attribute objects.
 /// \param Logger Pointer to spdlog instance to be used for logging.
 void writeArrayOfAttributes(hdf5::node::Node const &Node,
-                            const nlohmann::json &Values, SharedLogger Logger) {
+                            const nlohmann::json &Values,
+                            SharedLogger const &Logger) {
```

Comment:
East const, west const, east const -.-
Maybe we should standardise?

Reply:
East const for me

Reply:
That also seems to be most prevalent.

Reply:
Ok, fine with me.
I'll do my best to break my west const habit... I think Jack also picked up west const from me. Jonas uses east const though. Will try to remember to bring it up in a meeting.